src/embed/scene-info.js

```diff
@@ -47,7 +47,8 @@ SceneInfo.loadFromGetParams = function() {
   var params = {};
   for (var camelCase in CAMEL_TO_UNDERSCORE) {
     var underscore = CAMEL_TO_UNDERSCORE[camelCase];
-    params[camelCase] = Util.getQueryParameter(underscore);
+    params[camelCase] = Util.getQueryParameter(underscore)
+                        || window.WebVRConfig.PLAYER[underscore];
```

Comment:
This change breaks the code if there isn't a window.WebVRConfig attribute or if there isn't a window.WebVRConfig.PLAYER attribute.